tests/cisco_ios/show_interfaces_status/cisco_ios_show_interfaces_status2.raw

```diff
@@ -0,0 +1,20 @@
+
+--------------------------------------------------------------------------------
+Port          Name               Status    Vlan      Duplex  Speed   Type
+--------------------------------------------------------------------------------
+fc2/1 is up    Port mode is NP
+fc2/2 is up    Port mode is NP
+fc2/3 is up    Port mode is NP
+fc2/4 is up    Port mode is NP
+fc2/5 is down (SFP not present)
```

Comment:
We should make a new capture group for L9.

```
  ^\s*${PORT}\s+is\s+${STATUS}\s+\(${TYPE}\)$$
```

Reply:
Good catch, added a match for the FC DOWN.  Did not use TYPE as TYPE is the L1(Physical) interface/sfp type